#!/bin/sh /usr/share/dpatch/dpatch-run
## 304-magic-add-arm.dpatch by Russell Coker <russell@coker.com.au>
##
## DP: New magic for Adaptive Multi-Rate Codec files (Closes: #279322).

@DPATCH@

diff -Naur file-4.20.orig/magic/Magdir/audio file-4.20/magic/Magdir/audio
--- file-4.20.orig/magic/Magdir/audio	2007-04-03 16:17:31.000000000 +0000
+++ file-4.20/magic/Magdir/audio	2007-04-03 16:18:26.000000000 +0000
@@ -557,3 +557,8 @@
 # URL:  <http://www.monkeysaudio.com/>
 # From: Robert Millan <rmh@aybabtu.com>
 0	string	MAC\	Monkey's Audio file
+
+# Type: Adaptive Multi-Rate Codec
+# URL:  http://filext.com/detaillist.php?extdetail=AMR
+# From: Russell Coker <russell@coker.com.au>
+0	string	#!AMR	Adaptive Multi-Rate Codec (GSM telephony)
